docs/calibration.md

```diff
@@ -50,3 +76,20 @@ Manip mode urdf changes like so:
   </link>
   <link name="world"/>
 ```
+
+Just copy and paste that text into the file, and save. It should appear at the top level, inside the `<robot></robot>` tags.
+
+### Search and Replace
+
+We don't assume ROS on the server side, so there's one other piece of cleanup: replacing all meshes in your new modified urdf `stretch_manip_mode.urdf` with relative paths, instead of using ROS package management to find them.
+
+Use search and replace in your favorite editor to replace:
+```
+package://stretch_description
+```
+with
+```
+..
+```
+
+Then you should be able to run an example!
```

Comment:
Maybe give the command that helps you run that example?

Reply:
done